diff --git a/node_modules/react-native-music-control/android/src/main/java/com/tanguyantoine/react/MusicControlModule.java b/node_modules/react-native-music-control/android/src/main/java/com/tanguyantoine/react/MusicControlModule.java
index f33abc5..83ea8eb 100644
--- a/node_modules/react-native-music-control/android/src/main/java/com/tanguyantoine/react/MusicControlModule.java
+++ b/node_modules/react-native-music-control/android/src/main/java/com/tanguyantoine/react/MusicControlModule.java
@@ -201,7 +201,7 @@ public class MusicControlModule extends ReactContextBaseJavaModule implements Co
         filter.addAction(Intent.ACTION_MEDIA_BUTTON);
         filter.addAction(AudioManager.ACTION_AUDIO_BECOMING_NOISY);
         receiver = new MusicControlReceiver(this, context);
-        context.registerReceiver(receiver, filter);
+        ContextCompat.registerReceiver(context, receiver, filter, ContextCompat.RECEIVER_NOT_EXPORTED);
 
         Intent myIntent = new Intent(context, MusicControlNotification.NotificationService.class);
 
