diff --git a/node_modules/react-native-music-control/android/src/main/java/com/tanguyantoine/react/MusicControlModule.java b/node_modules/react-native-music-control/android/src/main/java/com/tanguyantoine/react/MusicControlModule.java
index f33abc5..9dd0c3a 100644
--- a/node_modules/react-native-music-control/android/src/main/java/com/tanguyantoine/react/MusicControlModule.java
+++ b/node_modules/react-native-music-control/android/src/main/java/com/tanguyantoine/react/MusicControlModule.java
@@ -201,7 +201,11 @@ public class MusicControlModule extends ReactContextBaseJavaModule implements Co
         filter.addAction(Intent.ACTION_MEDIA_BUTTON);
         filter.addAction(AudioManager.ACTION_AUDIO_BECOMING_NOISY);
         receiver = new MusicControlReceiver(this, context);
-        context.registerReceiver(receiver, filter);
+        if (Build.VERSION.SDK_INT >= 33) {
+            context.registerReceiver(receiver, filter, Context.RECEIVER_NOT_EXPORTED);
+        } else {
+            context.registerReceiver(receiver, filter);
+        }
 
         Intent myIntent = new Intent(context, MusicControlNotification.NotificationService.class);
 
